pom.xml

```diff
@@ -64,31 +64,31 @@
         <dependency>
             <groupId>org.slf4j</groupId>
             <artifactId>slf4j-api</artifactId>
-            <version>1.7.20</version>
+            <version>1.7.32</version>
         </dependency>
 
         <dependency>
             <groupId>org.slf4j</groupId>
             <artifactId>slf4j-ext</artifactId>
```

Comment:
why you doesn't use latest versions?

Reply:
I used closest version to decrease chance to break compatibility.

Reply:
It doesn't make sense since there are 12 version and 5 years of development between 1.7.20 and 1.7.32, and the version is 1.7.36. It is a minor version update, so no braking changes have to be here. Update to the latest one

Reply:
Done.